smartcontract/SwanFaucet.sol

```diff
@@ -37,4 +41,38 @@ contract SwanFaucet is OwnableUpgradeable {
         }
         return false;
     }
+
+    function sendTokensTo(address _address) public {
+        require(allowedToWithdraw(msg.sender), "please wait 24 hours");
+        nextAccessTime[_address] = block.timestamp + waitTime;
+
+        // transfer after setting internal state
+        tokenInstance.transfer(_address, tokenAmount);
+    }
+
+    modifier onlyAdmin {
+        require(isAdmin[msg.sender], "this sender is not an admin");
+        _;
+    }
+
+    function setAdmin(address _address) public onlyOwner {
+        isAdmin[_address] = true;
+    }
+
+    function removeAdmin(address _address) public onlyOwner {
+        isAdmin[_address] = false;
+    }
+
+    // pass in array of token addresses and the amount array to send, with a receiving address
+    function sendMultiToken(address[] memory _tokenAddresses, uint[] memory tokenAmounts, address _address) public onlyAdmin {
+        require(allowedToWithdraw(msg.sender), "please wait 24 hours");
+        
+        nextAccessTime[_address] = block.timestamp + waitTime;
+
+        for(uint i=0; i<_tokenAddresses.length; i++) {
+            require(tokenAmounts[i] <= ERC20(_tokenAddresses[i]).balanceOf(address(this)));
```

Comment:
can you change it to
```
if(tokenAmounts[i] <= ERC20(_tokenAddresses[i]).balanceOf(address(this)))
  ERC20(_tokenAddresses[i]).transfer(_address, tokenAmounts[i]);

```
Putting require(xx) here will stop transferring tokens which are enough to complete transfer